### ISSUE TITLE
Analyzer cannot find methods with parameters passed by readonly reference

### ISSUE THREAD

**Issue by tdeeb** [szehetner/InliningAnalyzer#19]
Version 0.7.8 of Inlining Analyzer fails to find many methods with parameters containing readonly references. This is a simple example:
```
public static int Clamp(in int value, in int min, in int max) => (value < min) ? min : (value > max) ? max : value;
```
If I remove the `in` modifiers, the method gets processed and it works fine. I'm using .NET Framework 4.5 with C# 7.3.

**Comment by szehetner** [comment]
Yes that was a bug. I just published a new version with this fix (0.7.9)

**Comment by tdeeb** [comment]
Thank you! The new version fixed the issue, but it still outputs messages saying the methods were not found.
